tests/test_processor.py

```diff
@@ -31,8 +34,11 @@ def test_Processor_suite_data_store_init_request(self):
         processors[Message.SuiteDataStoreInit](None, response, None)
 
         self.assertEqual(Message.ExecutionStatusResponse, response.messageType)
-        self.assertEqual(False, response.executionStatusResponse.executionResult.failed)
-        self.assertEqual(0, response.executionStatusResponse.executionResult.executionTime)
+        self.assertEqual(False,
```

Comment:
I am not sure if this line is required to be split. I use PyCharm's default formatting and it doesn't complain for lines having less than 120 chars. I think we can follow that convention. But I'd be happy to hear your thoughts on this.